packages/flutter/lib/src/material/text_form_field.dart

```diff
@@ -81,6 +81,12 @@ export 'package:flutter/services.dart' show SmartDashesType, SmartQuotesType;
 /// ** See code in examples/api/lib/material/text_form_field/text_form_field.1.dart **
 /// {@end-tool}
 ///
+/// {@tool dartpad}
+/// This example shows how to force error text to the field and cache it.
```

Comment:
I would add to the end: "after making an asynchronous call." or something like that.